### PR TITLE
Generate vk_dispatch_table_helper.h files using vk.xml

### PR DESCRIPTION
**Pull request by mark-lunarg**
Generating these header files was done via vk-generate.py based on parsing the header or hand-edited API metadata scripts.  Moved to using standard codegen.  Def files are next (will remove the vk-generate script after all functionality is migrated and trunk merges are complete).

### CODE REVIEW

File: scripts/dispatch_table_generator.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        self.protectFeature  = protectFeature
+        self.protectProto    = protectProto
+        self.protectProtoStr = protectProtoStr
+        self.apicall         = apicall
+        self.apientry        = apientry
+        self.apientryp       = apientryp
+        self.alignFuncParam  = alignFuncParam
+#
+# DispatchTableOutputGenerator - subclass of OutputGenerator.
+# Generates Vulkan API data for helper functions
```
Comment (mark-lunarg) [inline scripts/dispatch_table_generator.py]: Missed this comment -- will update.

File: build-android/android-generate.sh [hunk 2]
```diff
@@ -21,11 +21,10 @@ cd $dir
 rm -rf generated
 mkdir -p generated/include generated/common
 
-python ../scripts/vk-generate.py Android dispatch-table-ops layer > generated/include/vk_dispatch_table_helper.h
-
 python ../scripts/vk_helper.py --gen_enum_string_helper ../include/vulkan/vulkan.h --abs_out_dir generated/include
 python ../scripts/vk_helper.py --gen_struct_wrappers ../include/vulkan/vulkan.h --abs_out_dir generated/include
 
+( cd generated/include; python ../../../scripts/lvl_genvk.py -registry ../../../scripts/vk_dispatch_table_helper.h )
```
Comment (cnorthrop) [inline build-android/android-generate.sh]: This should be:

    ( cd generated/include; python ../../../scripts/lvl_genvk.py -registry ../../../scripts/vk.xml vk_dispatch_table_helper.h )

Comment (mark-lunarg) [inline build-android/android-generate.sh]: Yeah, sorry, fixed it here but forgot to push.  The patch currently runs cleanly on our build/test farm.

**Comment by mark-lunarg** [comment]
Cody verified, pushed to Master.

**Pull request closed without merging**